Remove python2-xml dependency from OpenPBS

jcsiadal opened the pull request:
Signed-off-by: jcsiadal <jeremy.c.siadal@intel.com>

OpenSUSE 15.4 removes Python 2 support. I can't see any reason for including this build dependency, as it still builds without it (artifact from old version?).

jcsiadal committed:
Remove Python2-xml dep

Signed-off-by: jcsiadal <jeremy.c.siadal@intel.com>

adrianreber commented:
Sounds good to me. I would just remove the line completely instead of commenting it out. It is tracked in git after all.

Concerning Python 2. There are still BuildRequires python2(-devel) in petsc and slepc. Have you tried to build that on OpenSUSE 15.4?

jcsiadal committed:
Remove instead of comment

Signed-off-by: jcsiadal <jeremy.c.siadal@intel.com>

jcsiadal commented:
Removed now. I'll be submitting PRs for those too, but I still need to confirm they work or update them.

adrianreber commented:
Could you rebase to pick up the latest CI changes? I would like to see if my changes actually works with a real PR.

jcsiadal commented:
Rebased to the current 2.x. The CI seems to be broken. See my gitignore PR, which does nothing besides updating the .gitignore file (I also rebased it to current 2.x).

adrianreber commented:
Thanks for the rebase. I am looking now into fixing the CI errors.

jcsiadal commented:
Rebased again. Passes validation checks now.

adrianreber commented:
Thanks for your patience with the new CI setup. I see the following message at the end of the wall of text:
```
Wrote: /github/home/rpmbuild/RPMS/x86_64/openpbs-server-ohpc-20.0.1-0.x86_64.rpm
Wrote: /github/home/rpmbuild/RPMS/x86_64/openpbs-execution-ohpc-20.0.1-0.x86_64.rpm
Wrote: /github/home/rpmbuild/RPMS/x86_64/openpbs-client-ohpc-20.0.1-0.x86_64.rpm
Wrote: /github/home/rpmbuild/RPMS/x86_64/openpbs-devel-ohpc-20.0.1-0.x86_64.rpm
```

So the CI was able to correctly detect which package you changed and the rebuild was successful.
